susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -131,7 +131,7 @@ install_gnupg_debian:
 {% include 'channels/gpg-keys.sls' %}
 
 salt-minion-package:
-  pkg.latest:
+  pkg.installed:
```

Comment:
@mcalmer I'm worried about how this could affect our side, where every machine has a salt-minion installed by sumaform, when configuring it after deployment. We will need to clean first those leftovers from sumaform, to avoid testing with an incorrect salt version?

We might have a similar behavior in our Build Validation pipelines, where we use MU repos to boostrap the minions.

Reply:
I think this is on the contrary an opportunity to clean up the leftovers from sumaform. It's a long time we want to do that and we already do it in BV testsuite. So I see it as a good thing.

I am just wondering why we should not do it in sumaform instead of doing it in test suite. This way, developers would benefit from the cleanup too. @rjmateus ping ?

Reply:
This time the return was quick. We got the positive feedback. We can delay merging some days if wanted.

Reply:
I have taken the liberty of triggering a full test suite of this PR https://ci.suse.de/view/Manager/view/Uyuni/job/uyuni-pr-ci-tests/55/

Reply:
CI test suite is not affected by this code change, so I guess we can merge it.

Reply:
Cool, thanks.